test/data.jl

```diff
@@ -1,8 +1,8 @@
 # Tests of data checking and transformation code
 
-function test_case(filename::AbstractString)
+function test_case(filename::AbstractString, parse_file::Function)
     temp_file = "temp.m"
-    source_data = PowerModels.parse_file(filename)        
+    source_data = parse_file(filename)        
```

Comment:
When I first implemented this Mile suggested that it would be good form, not to export `parse_file` to avoid collisions with other packages.

Reply:
We did not export `parse_file`. The `test_case` function takes it in as an argument. We test both `PowerModels.parse_file` and `PowerModels.parse_matpower` in `testset "test idempotent matpower export"`